src/app/view/components/listing-data-table/listing-data-table.component.ts

```diff
@@ -17,27 +18,38 @@ export class ListingDataTableComponent extends BaseComponent {
     @Input() columns: DataTableColumn[];
     @Input() url: string;
     @Input() actionHeaderOptions: ActionHeaderOptions;
+    @Input() buttonActions: ButtonAction[] = [];
+    @Input() defaultSortOrder: string;
+    @Input() defaultSortField: string;
     @Output() rowWasClicked: EventEmitter<any> = new EventEmitter;
 
     private paginatorRows: number;
     private paginatorLinks: number;
     private items: any[];
     private totalRecords: number;
     private query = '';
-    // tslint:disable-next-line:no-unused-variable
+    private sortField: string;
     private selectedItems = [];
+    private dateColumns: DataTableColumn[];
 
     constructor(private dotcmsConfig: DotcmsConfig, private crudService: CrudService,
-    messageService: MessageService, public loggerService: LoggerService) {
+    messageService: MessageService, public loggerService: LoggerService, private formatDateService: FormatDateService) {
         super(['global-search'], messageService);
     }
-
     ngOnInit(): void {
         this.dotcmsConfig.getConfig().subscribe(configParams => {
             this.paginatorRows = configParams.paginatorRows;
             this.paginatorLinks = configParams.paginatorLinks;
             this.loadData(this.paginatorRows, -1);
         });
+
+        this.sortField = this.defaultSortField;
+    }
+
+    ngOnChanges(changes): void {
+        if (changes.columns) {
+            this.dateColumns = changes.columns.currentValue.filter(column => column.format === 'date');
```

Comment:
Here we can use a read only value, something like:

readonly DATE_FORMAT = 'date';

and ...
...
this.dateColumns = changes.columns.currentValue.filter(column => column.format === this.DATE_FORMAT);
...

Also, I dont know if handle this property as a String is a good idea, do we expect others values for this property in the future?, I don't think so, so I think we should use a boolean for it and call it something like 'dateFormat' (at least for now)